tidb-cloud/connect-tidbcloud-to-vercel.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Integrate TiDB Cloud with Vercel
+summary: Learn how to connect your TiDB Cloud clusters to Vercel projects.
+---
+
+# Integrate TiDB Cloud with Vercel
+
+[Vercel](https://vercel.com/) is the platform for frontend developers, providing the speed and reliability innovators need to create at the moment of inspiration.
+
+This guide introduces how to connect your TiDB Cloud clusters to Vercel projects using one of the following methods:
+
+* [Connect via the TiDB Cloud Vercel Integration](#connect-via-the-tidb-cloud-vercel-integration)
+* [Connect via manually configuring environment variables](#connect-via-manually-setting-environment-variables)
+
+## Prerequisites
+
+Before connection, make sure the following prerequisites are met.
+
+### Available Vercel account and project
+
+You are expected to have an account and a project in Vercel. If you do not have any, refer to the following Vercel documents to create one:
+
+* [Creating a new personal account](https://vercel.com/docs/teams-and-accounts#creating-a-personal-account) or [Creating a new team](https://vercel.com/docs/teams-and-accounts/create-or-join-a-team#creating-a-team).
+* [Creating a project](https://vercel.com/docs/concepts/projects/overview#creating-a-project) in Vercel, or if you do not have an application to deploy, you can use the [TiDB Cloud Starter Template](https://vercel.com/templates/next.js/tidb-cloud-starter) to have a try.
+
+One Vercel project can only connect to one TiDB Cloud cluster. To change the integration, you need to first disconnect the current cluster and then connect to a new cluster.
+
+### Available TiDB Cloud account and cluster
+
+You are expected to have an account and a cluster in TiDB Cloud. If you do not have any, refer to [Create a TiDB cluster](/tidb-cloud/create-tidb-cluster.md) in TiDB Cloud.
+
+To integrate with Vercel, you are expected to have the "Organization Owner" access to your organization or "Organization Member" + "Project" access to the target project in TiDB Cloud.
+
+One TiDB Cloud cluster can connect to multiple Vercel projects.
+
+### All IP addresses allowed for traffic filter in TiDB Cloud
+
+Make sure that the traffic filter of your TiDB Cloud cluster allows all IP addresses (set to `0.0.0.0/0`) for connection, this is because Vercel deployments use [dynamic IP  addresses](https://vercel.com/guides/how-to-allowlist-deployment-ip-address). If you use TiDB Cloud Vercel Integration, TiDB Cloud automatically adds a `0.0.0.0/0` traffic filter to your cluster in the integration workflow if there is none.
```

Comment:
```suggestion
Make sure that the traffic filter of your TiDB Cloud cluster allows all IP addresses (set to `0.0.0.0/0`) for connection, this is because Vercel deployments use [dynamic IP  addresses](https://vercel.com/guides/how-to-allowlist-deployment-ip-address). If you use the TiDB Cloud Vercel integration, TiDB Cloud automatically adds a `0.0.0.0/0` traffic filter to your cluster in the integration workflow if there is none.
```